packages/mobile/src/navigator/NavigationService.ts

```diff
@@ -50,12 +53,47 @@ export function navigate(routeName: string, params?: NavigationParams) {
     })
 }
 
-export function navigateAfterPinEntered(routeName: string, params?: NavigationParams) {
-  navigate('Background', {
-    onUnlock() {
-      navigate(routeName, params)
-    },
-  })
+export async function ensurePincode(): Promise<boolean> {
```

Comment:
Not a big deal but I would expect this code to live next to other pin related logic